client/views/actions/nick.tpl

```diff
@@ -1,3 +1,3 @@
 {{> ../user_name nick=from}}
-is now known as
+{{translate 'client.now_known_as'}}
 {{> ../user_name nick=new_nick}}
```

Comment:
Actually, not sure how to solve this exactly, because it requires passing a template into a template...